cwl/tools/merge_sv.cwl

```diff
@@ -0,0 +1,26 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: merge_sv
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: DockerRequirement
+    dockerPull: 'zhangb1/data-assembly'
```

Comment:
any plans to use the sbg repo instead of dockerhub?

Reply:
now will be only on dockerhub, maybe move to sbg , not sure yet